docs/exposing-binding-data/olm-descriptors.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 5
+---
+
+# Adding Binding Metadata as OLM Descriptors
+
+Use this method if your Operator is distributed as an [Operator
+Bundle](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md).
+You can add [OLM
+descriptors](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md)
+to describe about the binding metadata are that is to be exposed.  The OLM
```

Comment:
- binding metadata are that is to be exposed ---> binding metadata that is to be exposed

Reply:
- to describe about the binding metadata ---> to describe the binding metadata

Reply:
Done. be8f414b3ce89813f5286cd716802547d9f5e86b